listenbrainz/webserver/static/js/jsx/spotify-player.jsx

```diff
@@ -89,11 +89,13 @@ export class SpotifyPlayer extends React.Component {
         </span>, "Found a match");
         return this.play_spotify_uri(track.uri);
       }
-      return this.handleWarning("Could not find track on Spotify");
+      this.handleWarning("Could not find track on Spotify");
+      this.playNextTrack();
```

Comment:
Not returning anything here?

Reply:
No need since it's the last link in our promise chain. We don't have anything to return, and it's not strictly needed to return anything (https://stackoverflow.com/questions/53499297/is-the-return-type-of-then-a-promise-even-with-no-return-statement#comment93867901_53499297)

Reply:
Alright, makes sense.